core/src/lib/parse_conf.h

```diff
@@ -353,8 +353,9 @@ class ConfigurationParser {
   BStringList warnings_;
 
 
-  const char* GetDefaultConfigDir();
-  bool GetConfigFile(PoolMem& full_path,
+  public: static const char* GetDefaultConfigDir();
```

Comment:
public: needs to go into an own line. Please format the code with bareos-checks-sources.